express/blocks/submit-email/submit-email.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+function decorateSubmitEmailBlock($block) {
+  const container = document.querySelector('.submit-email-container');
+
+  const icon = container.querySelector('.icon-creative-cloud-express, .icon-cc-express-logo');
```

Comment:
This confused me.. sometimes the class on the image was `icon-creative-cloud-express` and sometimes it was `icon-cc-express-logo`?

Reply:
@dylandepass I think I see where the confusion is coming from: On the homepage, it's defined like this:
<img width="580" alt="image" src="https://user-images.githubusercontent.com/1609742/154821322-426862e7-3e48-4309-8115-eb5483d036c5.png">
So if the viewport is smaller than 400px or so, it will pick the mobile icon vs the regular one (a feature I had already forgotten we built)...

Does that make sense?

Reply:
@rofe That does, thanks for clearing that up. Here is the the part that messed me up.

If you right click the image and select `Alt Text` then you get a modal that show's there is nothing there.
![Screen Shot 2022-02-21 at 11 26 55 AM](https://user-images.githubusercontent.com/3231084/154994401-24c15a90-f5b4-47ce-98e3-e8d4356e2cff.png)

Looks like you have to use the `Alt Text` button in the top menu bar under `Picture` in order to see that alt text has been added to the imagine. 🤔 🤷🏽‍♂️

Reply:
Correct. I wasn't aware of the right-click way to set the alt text, and it's very surprising that it doesn't use the same properties underneath. We should mention that in the FAQ.